<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>CharacterSelectWindow</class>
 <widget class="QMainWindow" name="CharacterSelectWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>600</width>
    <height>400</height>
   </rect>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <widget class="QComboBox" name="comboBox">
      <item>
       <property name="text">
        <string>调光开关</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>Ezreal</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>Lux</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>Garen</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>Teemo</string>
       </property>
      </item>
     </widget>
    </item>
    <item>
     <widget class="QLabel" name="characterInfoLabel">
      <property name="text">
       <string>Select a character to view information</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="confirm_button">
      <property name="text">
       <string>Confirm Selection</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
</ui>
